### ISSUE TITLE
[Concept Entry] TensorFlow keras

### ISSUE THREAD

**Issue by codecademy-docs** [Codecademy/docs#4593]
### Reason/inspiration (optional)

We would like a new entry on the `Keras` concept under TensorFlow. The entry should go in a new file under `docs/content/tensorflow/concepts/keras/keras.md`.

The entry should include:

- An introductory paragraph that describes the concept
- Explain the keras module in tensorflow

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

**Comment by larbyke** [comment]
I found out about documents through friend

**Comment by mamtawardhani** [comment]
Hey @larbyke! You’re assigned 🎉 In addition to the documents linked in the description, please also look at the [Contribution Guide](https://www.codecademy.com/resources/docs/contribution-guide). After creating a PR, the maintainer(s) (with the collaborator label) will add comments/suggestions to address any revisions before approval.

Is this your first contribution to Codecademy Docs? If so, we’re curious to know how you found out about contributing to Docs.

**Comment by mamtawardhani** [comment]
Unassigning the issue due to the inactivity of @larbyke
